src/main/java/com/enterprise/redcord/dao/ITopicDAO.java

```diff
@@ -1,9 +1,14 @@
 package com.enterprise.redcord.dao;
 
+import com.enterprise.redcord.dto.Message;
 import com.enterprise.redcord.dto.Topic;
 
+import java.util.List;
+
 public interface ITopicDAO {
 
-    Topic save(Topic topic) throws Exception;
```

Comment:
I decided to change the name of the save method in the ITopicDAO to saveTopic to give it a little more description so its no confusing when calling it/using it in other parts of code since there are similar methods such as saveMessage. save and saveMessage seem a little too hard to differentiate and saveTopic can easily clear up that misunderstanding.

Naming conventions: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3 
Reduce technical debt: https://martinfowler.com/bliki/TechnicalDebt.html